alcs-api/src/providers/typeorm/migrations/1662659560119-fix_duplicate_board_status.ts

```diff
@@ -0,0 +1,43 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class fixDuplicateBoardStatus1662659560119
+  implements MigrationInterface
+{
+  name = 'fixDuplicateBoardStatus1662659560119';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    //Delete existing to clear duplicates
+    await queryRunner.query(
+      `DELETE FROM "board_status" WHERE board_uuid = 'd8c18278-cb41-474e-a180-534a101243ab'`,
+    );
+
+    //Re-insert a single set
+    await queryRunner.query(`
+      INSERT INTO "public"."board_status"
+      ("uuid", "audit_deleted_date_at", "audit_created_at", "audit_updated_at", "audit_created_by", "audit_updated_by", "order", "board_uuid", "status_uuid") VALUES
+       --- Executive Committee
+      (DEFAULT, DEFAULT, DEFAULT, DEFAULT, 'alcs-api', DEFAULT, 0, 'd8c18278-cb41-474e-a180-534a101243ab', 'e6ddd1af-1cb9-4e45-962a-92e8d532b149'),
```

Comment:
Are all the board UUIDs pregenerated via migrations?

Reply:
Yes! We pre-generated them all so its the same across all systems to keep things easy like this script.